sycl/unittests/assert/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_sycl_unittest(AssertTests OBJECT
```

Comment:
NIT. It doesn't look reasonable to have a separate executable for just assert. Are there plans to extend this suite with more tests in future?

Reply:
There's some failure when multiple `PiImageArray` instances are defined within single application. I'm going to investigate it eventually later. Hence, the assert unit-test will reside in a distinct executable within this patch.